tests/diagramconfiguration_test.py

```diff
@@ -1,12 +1,12 @@
-from INCode.models import CompilationDatabases, Index, Callable
+from INCode.models import Callable
 from INCode.diagramconfiguration import DiagramConfiguration, CallableTreeItem
-from test_environment_generation import *
+from tests.test_environment_generation import *
```

Comment:
Wild-card imports are not recommended by [PEP 8](https://www.python.org/dev/peps/pep-0008/#imports).